packages/superset-ui-preset-chart-xy/types/@vx/legend/index.d.ts

```diff
@@ -2,15 +2,15 @@
 declare module '@vx/legend' {
   import React from 'react';
 
-  export function LegendOrdinal(props: { [key: string]: any }): React.ReactNode;
+  export function LegendOrdinal(props: { [key: string]: any }): React.ReactElement;
```

Comment:
didn't like `ReactNode`? (you can ignore/resolve my other suggestions of `ReactNode` if they don't work, was mostly trying to identify spots where we deviate from `Node`)

Reply:
![image](https://user-images.githubusercontent.com/1659771/55052600-c79d7b00-5016-11e9-96e8-72da9007323a.png)

Reply:
These require `ReactElement` because they expect a constructor. `ReactNode` is more suitable for child which can be `string | number | null | undefined`